unit-tests-jest/numbers.js

```diff
@@ -9,7 +9,7 @@ export function evenNumbers(numbers) {
  * Returns a number formatted in dollars and cents.
  */
 export function toDollars(amount) {
-  return `$${amount}.00`;
+  return `$${amount}`;
```

Comment:
Still not right. It should format to dollars and cents. Try formatting `12.4335`